igor-web/src/main/groovy/com/netflix/spinnaker/igor/service/BuildOperations.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 Schibsted ASA.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.service;
+
+import com.netflix.spinnaker.igor.build.model.GenericBuild;
+import com.netflix.spinnaker.igor.build.model.GenericGitRevision;
+
+import java.util.List;
+import java.util.Map;
+
+public interface BuildOperations extends BuildService {
```

Comment:
I'd like to see class- & method-level docs for all interfaces introduced into the Spinnaker ecosystem.

Reply:
Better?